lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/handler/CheckPassportHandler.java

```diff
@@ -269,7 +270,7 @@ private APIGatewayProxyResponseEvent lambdaCompletedOK(
 
     private PassportFormData parsePassportFormRequest(String input)
             throws OAuthHttpResponseExceptionWithErrorBody {
-        LOGGER.info("Parsing passport form data into payload for DCS");
+        LOGGER.info("Parsing passport form data into payload");
```

Comment:
Could this be for third party document check 

Just keeps it really clear

Reply:
Reworded.